include/wifi_hal.h

```diff
@@ -103,9 +102,17 @@
  */
 #include "wifi_hal_telemetry.h"
 /**
+* Wifi client management specific APIs.
+ */
+#include "wifi_hal_client_mgt.h"
+/**
 * Wifi extender specific APIs.
 * Includes API's used by Plume Mesh.
  */
 #include "wifi_hal_extender.h"
-
+/**
+* Wifi deprecated APIs.
+* APIs to be deprecated. Not add new function or structure!
+*/
+#include "wifi_hal_deprecated.h"
```

Comment:
wifi_hal_deprecated.h is required as rdk-wifi-hal has functionality which is using the structure referred in wifi_hal_deprecated.h. Without that, there are multiple compilation issues noticed.

Reply:
why is it called deprecated.h then, because it was completely removed in the previous release.

Reply:
This is needed for now as there are structures in this file being used without which compilation issue are being noticed. I will check further with the team on the history of this file which I am not much aware. Please add it back in the meantime.